# information about frame and timestamp
Header header

# neatness
float32 neatness
float32 group_neatness
float32 filling_neatness
float32 pulling_neatness